aws-ass-start.py

```diff
@@ -369,6 +369,38 @@ def create_deleted_tagged_beanstalk_environments(cfg, aws):
     cfg.get_logger().info(f"Creation of terminated BeanStalk environments ended")
 
 
+def restore_s3_backup(cfg, aws):
+    s3_client = aws.get_boto3_client('s3')
+
+    try:
+        cfg.get_logger().info("Start getting bucket names")
+        response = s3_client.list_buckets()
+        s3_list = response['Buckets']
+        cfg.get_logger().debug(response)
+        cfg.get_logger().debug(s3_list)
+        cfg.get_logger().info("Getting bucket names finished successfully")
+        for bucket in s3_list:
+            bucket_name = bucket['Name']
+            bucket_arn = f"arn:aws:s3:::{bucket_name}"
+            cfg.get_logger().debug(f"Checking bucket {bucket_name} ({bucket_arn})")
+            if aws.s3_has_tag(bucket_name, cfg.full_ass_tag("ass:s3:clean-bucket-on-stop"), "yes") or aws.s3_has_tag(bucket_name, cfg.full_ass_tag("ass:s3:backup-and-empty-bucket-on-stop"), "yes"):
```

Comment:
No need to check for `ass:s3:clean-bucket-on-stop` here, since you do not (and should not) take a backup of the buckets with this tag.